eduvpn/utils.py

```diff
@@ -30,6 +30,10 @@ def get_prefix() -> str:
     raise Exception("Can't find eduVPN installation")
 
 
+def get_config_dir() -> str:
+    return environ.get("XDG_CONFIG_HOME", "$HOME/.config")
```

Comment:
is `$HOME` automatically expanded by `environ.get()`? otherwise, I think we should  use `environ.get('HOME', '~')`

Reply:
I wrongly assumed it would be expanded by expanduser, as `~` is equal to `$HOME`, but it only expands `~`. Changed it to just default to `~/.config` instead.